src/config.rs

```diff
@@ -137,16 +137,28 @@ pub enum Command {
 impl DefGuardConfig {
     pub fn new() -> Self {
         let mut config = Self::parse();
-        if config.webauthn_rp_id.is_none() {
-            config.webauthn_rp_id = Some(
-                config
-                    .url
+        config.validate_rp_id();
+        config
+    }
+
+    // this is an ugly workaround to avoid `cargo test` args being capture by clap
+    pub fn new_test_config() -> Self {
+        let mut config = Self::parse_from::<Vec<_>, String>(vec![]);
```

Comment:
To nie zadziała? `[]` też jest iteratorem.
```suggestion
        let mut config = Self::parse_from([]);
```